modules/nf-core/pypgx/runngspipeline/main.nf

```diff
@@ -0,0 +1,58 @@
+process PYPGX_RUNNGSPIPELINE {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/pypgx:0.25.0--pyh7e72e81_0':
+        'biocontainers/pypgx:0.25.0--pyh7e72e81_0' }"
+
+    input:
+    tuple val(meta), path(vcf), path(tbi), path(coverage), path(control_stats), val(pgx_gene)
+    tuple val(meta2), path(resource_bundle)
+    val(assembly_version)
+
+    output:
+    tuple val(meta), val(pgx_gene), path("*pypgx_output"), emit: outdir
```

Comment:
I don't think that the `val(pgx_gene)` should be here.
What is the rational behind this choice ?